web/packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -103,3 +102,12 @@ export default function useNodes({ clusterId, id }: stores.DocumentNodes) {
     ...paginationProps,
   };
 }
+
+// sort logins by making 'root' as the first in the list
+export const sortLogins = (logins: string[]) => {
+  const noRoot = logins.filter(l => l !== 'root').sort();
```

Comment:
Nit: I usually avoid using `sort()` this way, even if it's totally correct, since it creates an illusion that it's a pure function and may invite people to accidentally introduce buggy behavior later. This way is more explicit:

```
const noRoot = logins.filter(l => l !== 'root');
noRoot.sort();
```